SwaprHQ/swapr-dapp
# [SWA-142][IMPROVEMENT] - Update Kleros token list URL

## Fixes: [SWA-142](https://linear.app/swaprhq/issue/SWA-142/update-kleros-token-list-url)

# Description

* Update Kleros token list URL

# Visual Evidence
![image](https://github.com/SwaprHQ/swapr-dapp/assets/21271189/0ca2d62e-0113-475d-a65c-ae840e9b30f5)


# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr dapp page
4) Connect your wallet
5) Go to the Swapbox, select a token, click on "Manage token list" and scroll down, you should see Kleros token list and be able to toggle it on
